source/deploy/mattermost-changelog.md

```diff
@@ -11,6 +11,110 @@ From Mattermost v9.2, this changelog summarizes updates for the latest cloud and
+ - Exposed ``UpdateUserRoles`` in ``pluginapi``.
+ - Exposed ``pluginapi.ProfileImageBytes`` to simplify bot setup from a plugin.
+ - For ``POST /channels``, added a validation for ``display_name`` to not pass validation if the display name is empty.
+
+### Bug Fixes
+ - Fixed an issue with context cancellation for integration requests.
+ - Fixed an issue preventing the retrieval of SAML metadata.
+ - Fixed an issue causing an empty channel switcher after converting a Group Message to a private channel.
+ - Fixed an issue where System Admins were not allowed to LDAP sync SAML users when ``SamlSettings.EnableSyncWithLdap`` was true.
+ - Fixed an issue with markdown in the AD job status table.
+ - Fixed an issue with a control character in the group list modal.
+ - Fixed an issue where the auto-complete channels API returned archived channels in response.
+ - Fixed a crash issue in the **System Console**.
+ - Fixed an issue where links included in notifications were truncated and not clickable.
+ - Fixed using local requests instead of HTTP requests in the flow library.
+ - Fixed an issue where ``support_packet.yaml`` wasn’t generated even if an error occurred.
+ - Fixed an issue where outgoing webhooks did not trigger when using multiple callback URLs.
+ - Fixed an issue where it was not possible to clear plugin settings with a default value in the **System Console**.
+ - Fixed an issue where ``MaxUsersForStatistics`` wasn’t ignored when generating a Support Packet.
+ - Fixed an issue where the ``EnsureBot`` function did not recreate the bot if it had been manually deleted.
+ - Fixed an issue where users couldn't look up a user by their ID in the **System Console** anymore.
+ - Fixed an accessibility issue where the focus didn’t go back to the originating button when a modal was closed.
+ - Fixed an issue where end users were not allowed to fetch the group members list of groups that allow ``@-mentions``.
+
+### config.json
+New setting option were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
+
+#### Changes to all plans:
+ - Under ``FileSettings`` in ``config.json``:
+   - Added ``AmazonS3UploadPartSizeBytes`` and ``ExportAmazonS3UploadPartSizeBytes`` to control the part size used to upload files to an S3 store.
+ - Under ``ServiceSettings`` in ``config.json``:
+   - Increased the default payload size limit (``MaximumPayloadSizeBytes``) from 100 kB to 300 kB.
+ - Under ``ClusterSettings`` in ``config.json``:
+   - Removed unused settings ``StreamingPort``, ``MaxIdleConns``, ``MaxIdleConnsPerHost`` and ``IdleConnTimeoutMilliseconds``.
+
+ #### Changes to Professional and Enterprise plans:
+ - Under ``ExperimentalSettings`` in ``config.json``:
+   - Removed the ``UseNewSAMLLibrary`` experimental setting.
+
+### Go Version
+ - v9.8 is built with Go ``v1.21.0``.
```

Comment:
1.21.8. Can you clarify from where you found that it's 1.21.0? We should probably get it fixed.

Reply:
https://github.com/mattermost/mattermost/pull/26512 - dot release version wasn't mentioned in the release note.

Reply:
Oh okay. I thought we agreed to check the `server/build/Dockerfile.buildenv` file for the exact version. But anyways, my bad on the release notes.

Reply:
I forgot to, thank you 👍